services/QuillLMS/app/controllers/api/v1/feedback_history_ratings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::FeedbackHistoryRatingsController < ApplicationController
+  before_action :set_feedback_history_rating, only: [:show, :edit, :update, :destroy]
+  skip_before_action :verify_authenticity_token
```

Comment:
From the [specification](https://api.rubyonrails.org/classes/ActionController/RequestForgeryProtection.html): `APIs may want to disable this behavior since they are typically designed to be state-less: that is, the request API client handles the session instead of Rails.` Since you raise this point, I will move this line into the parent controller (unless it breaks a bunch of stuff)